mne/io/egi/tests/test_egi.py

```diff
@@ -586,3 +586,17 @@ def test_set_standard_montage_mff(fname, standard_montage):
     # Check that the reference remained
     for pick in picks:
         assert_allclose(raw.info["chs"][pick]["loc"][3:6], ref_loc)
+
+
+@requires_testing_data
+def test_egi_mff_bad_xml(tmp_path):
+    """Test that corrupt XML files are gracefully handled."""
+    pytest.importorskip("defusedxml")
+    mff_fname = shutil.copytree(egi_mff_fname, tmp_path / "test_egi_bad_xml.mff")
+    bad_xml = mff_fname / "bad.xml"
+    with bad_xml.open("w") as fid:
+        fid.write("<foo>")
```

Comment:
If you want to venture into bonus territory you can save a line with Path's nice method `write_text`:
```suggestion
    bad_xml = mff_fname / "bad.xml"
    bad_xml.write_text("<foo>", codec="utf-8")
```